Src/AutoFakeItEasy2/FakeItEasyBuilder.cs

```diff
@@ -56,7 +56,7 @@ public ISpecimenBuilder Builder
         /// The Create method checks whether a request is for an interface or abstract class. If so
         /// it delegates the call to the decorated <see cref="Builder"/>. When the specimen is
         /// returned from the decorated <see cref="ISpecimenBuilder"/> the method checks whether
-        /// the returned instance is a FakeItEasy instance.
```

Comment:
Could we tackle *AutoFixture.AutoFakeItEasy2* on a separate Pull Request? :smiley: